python_modules/libraries/dagster-dbt/dagster_dbt/rpc/resources.py

```diff
@@ -440,6 +439,75 @@ def dbt_rpc_pipeline():
     return DbtRpcClient(host=context.resource_config["host"], port=context.resource_config["port"])
 
 
+class DbtRpcSyncClient(DbtRpcClient):
+    def _get_result(self, data: str = None) -> DbtRpcOutput:
+        """Sends a request to the dbt RPC server and continuously polls for the status of a request until the state is ``success``."""
+
+        out = super()._get_result(data)
+        request_token = out.result.get("request_token")
+
+        logs_start = 0
+        interval = 1
```

Comment:
Does the interval need to be configurable for the user? Currently it is not a param of this function. Additionally, does this function need to include logging?

Reply:
While it's not a hard requirement, I do think it would be nice to allow for users to change this. Rather than an argument to the function, I think it'd make sense for this to be a property of the class  that gets set on initialization through some configuration option.